api/metrics_service.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package envoy.api.v2;
+
+import "google/api/annotations.proto";
+import "metrics.proto";
+
+// Service for streaming metrics to connected end point.
+service MetricsService {
```

Comment:
Does it make sense to explicitly call out PrometheusMetricsService ? After all the format is for Prometheus. If not this, atleast have a type field or something in streammessage that says what type of metric is coming out of Envoy.

Reply:
I think we discussed it in the PR https://github.com/envoyproxy/data-plane-api/pull/210 that we do not want to call this PrometheusMetricsService. We picked Promotheus model because it is more comprehensive representation of metrics model. We tried to look at OpenMetrics but since it is far off and any way is heavily inspired from Promotheus model. So we decided to go with this model for now.